### ISSUE TITLE
[ShadowFormer] Unnecessary `Transpose` extrapolated immediately after `Softmax`

### ISSUE THREAD

**Issue by PINTO0309** [PINTO0309/onnx2tf#205]
### Issue Type

Others

### onnx2tf version number

1.7.6

### onnx version number

1.13.1

### tensorflow version number

2.12.0rc0

### Download URL for ONNX

https://github.com/PINTO0309/PINTO_model_zoo/tree/main/353_ShadowFormer

### Parameter Replacement JSON

```json
{
  "format_version": 1,
  "operations": [
    {
      "op_name": "/encoderlayer_2/blocks.0/attn/Unsqueeze",
      "param_target": "op",
      "param_name": "/encoderlayer_2/blocks.0/attn/Unsqueeze_output_0",
      "new_shape": [24,1,100,100]
    },
    {
      "op_name": "/encoderlayer_2/blocks.1/attn/Unsqueeze",
      "param_target": "op",
      "param_name": "/encoderlayer_2/blocks.1/attn/Unsqueeze_output_0",
      "new_shape": [24,1,100,100]
    },
    {
      "op_name": "/conv/blocks.0/attn/Unsqueeze",
      "param_target": "op",
      "param_name": "/conv/blocks.0/attn/Unsqueeze_output_0",
      "new_shape": [6,1,100,100]
    },
    {
      "op_name": "/conv/blocks.1/attn/Unsqueeze",
      "param_target": "op",
      "param_name": "/conv/blocks.1/attn/Unsqueeze_output_0",
      "new_shape": [6,1,100,100]
    }
  ]
}
```

### Description

1. Personal
2. Unnecessary `Transpose` extrapolated immediately after `Softmax`. #180 found a pattern of shortcomings in the response. If onnx and tensorflow have the same input geometry and the same axis, we need to extrapolate `Transpose` immediately after Softmax to counteract the `Transpose` that tensorflow extrapolates on its own.
    ![image](https://user-images.githubusercontent.com/33194443/221242779-bc1629d8-9e1b-44d0-97a0-506a9bb9c218.png)
5. Ref: [[FastestDet] height and width axes are switched before last concatenation #180](https://github.com/PINTO0309/onnx2tf/issues/180)

**Comment by PINTO0309** [comment]
Fixes.
![image](https://user-images.githubusercontent.com/33194443/221334207-8fbaf503-4cba-4b12-9328-faca4827d915.png)
https://github.com/PINTO0309/onnx2tf/releases/tag/1.7.7
